public/scripts/backlog.js

```diff
@@ -0,0 +1,21 @@
+function getBacklog() {
+    var result = "ok";
+    db.collection("Backlog")
+        .onSnapshot(function(snapshot) {
+            snapshot.docChanges().forEach(function(change) {
+                if (change.type === "added") {
+                    console.log('added');
+                    createInstance(change.doc.data());
+                }
+                if (change.type === "modified") {
+                    console.log('modified');
+                    updateInstance(change.doc.data());
+                }
+                if (change.type === "removed") {
+                    console.log('removed');
+                    removeInstance(change.doc.data());
+                }
+            });
+        });
+    return result;
+}
```

Comment:
Similar method is already available we don't need this to be written again, make modification in old one, a small change is needed just think how.

Reply:
ok